docs/sphinx/user_guide/feature/loop_basic.rst

```diff
@@ -53,28 +53,42 @@ This may be written using ``RAJA::forall`` as::
 A ``RAJA::forall`` loop execution method is a template that takes an
 *execution policy* type template parameter. A ``RAJA::forall`` method takes
 two arguments: an iteration space object, such as a contiguous range of loop
-indices as shown here, and a single lambda expression representing the loop 
+indices as shown here, and a single lambda expression representing the loop
 kernel body.
 
-Applying different loop execution policies enables the loop to run in 
-different ways; e.g., using different programming model back-ends. Different 
-iteration space objects enable the loop iterates to be partitioned, reordered, 
+Applying different loop execution policies enables the loop to run in
+different ways; e.g., using different programming model back-ends. Different
+iteration space objects enable the loop iterates to be partitioned, reordered,
 run in different threads, etc. Please see :ref:`feat-index-label` for details
-about RAJA iteration spaces. 
+about RAJA iteration spaces.
 
 .. note:: Changing loop execution policy types and iteration space constructs
-          enables loops to run in different ways by recompiling the code and 
+          enables loops to run in different ways by recompiling the code and
           without modifying the loop kernel code.
 
-While loop execution using ``RAJA::forall`` methods is a subset of 
-``RAJA::kernel`` functionality, described next, we maintain the 
-``RAJA::forall`` interface for simple loop execution because the syntax is 
+As an extension of ``RAJA::forall``, the ``RAJA::expt::dynamic_forall`` method enables users
+to compile using a list of execution policies and choose the execution policy at run-time.
+For example, a user may want to have N policies available and at run-time choose which policy to use::
+
+  using exec_pol_list = camp::list<pol_0,
+				    ...
+				   pol_N>;
+  int pol = i; //run-time value
+
+  RAJA::expt::dynamic_forall<exec_pol_list>(pol, RAJA::TypedRangeSegment<int>(0, N), [=] (int i) {
+     c[i]  = a[i] + b[i];
+  });
+
+
+While loop execution using ``RAJA::forall`` methods is a subset of
```

Comment:
Confirming that this statement **does not** apply to RAJA::expt::dynamic_forall.

Reply:
I think we can include ``RAJA::dynamic_forall`` in the paragraph, we would just have to note that kernel does not support dynamic policy selection.

Reply:
works for me.